### ISSUE TITLE
More prominence, please, to the shortlist of extensions that may conflict with Simple Tab Groups

### ISSUE THREAD

**Issue by grahamperrin** [Drive4ik/simple-tab-groups#781]
From https://github.com/Drive4ik/simple-tab-groups/issues/545#issuecomment-594655891 (2020-03-04)

>>> … making a list of conflicting addons to notify the user if he has one …

🏆

Noted with thanks, at <https://addons.mozilla.org/addon/simple-tab-groups/>: 

>> … This extension may conflict with other programs similar in functionality.
>> 
>> Conflicted extensions include, but may not be limited to: … etc.

The current shortlist from AMO, with links added by me: 

* [OneTab](https://addons.mozilla.org/addon/onetab/)
* [Panorama Tab Groups](https://addons.mozilla.org/addon/panorama-tab-groups/)
* [Panorama View](https://addons.mozilla.org/addon/panorama-view/)
* [Tab Open/Close Control](https://addons.mozilla.org/addon/tab-open-close-control/)
* [Tiled Tab Groups](https://addons.mozilla.org/addon/tiled-tab-groups/)
* [Totally not Panorama (Tab Groups with tab hiding)](https://addons.mozilla.org/addon/basic-panorama/)

From <https://github.com/Drive4ik/simple-tab-groups/issues/716#issuecomment-782888817>: 

> … Can you make … extension conflicts bold or somewhat more visible in the installation instructions perhaps? …

**Comment by grahamperrin** [comment]
### Tentatively

Six more: 

* https://github.com/Drive4ik/simple-tab-groups/issues/779#issuecomment-785536590

Tentative, because whilst each one of the six conflicts is consistently reproducible with a release _candidate_ of Firefox `86.0` on [Tier-3](https://firefox-source-docs.mozilla.org/build/buildsystem/supported-configurations.html#tier-3) FreeBSD `14.0-CURRENT`, I have not yet tested with a Tier-1 release this version of Firefox.

**Comment by grahamperrin** [comment]
#393 (2019-05-26)

> … SmartAdBlock …

https://github.com/Drive4ik/simple-tab-groups/issues/533#issuecomment-590494799 (2020-02-14)

> … Mate Translate …

**Comment by Mek101** [comment]
Could you add the option to disable the conflict warning popup tab? I'm also using OneTab (while being mindful of ordering the tabs so it catches only the ones in the current group), and since the last week I keep having the warning tab pop up randomly and disrupt my work.
Thanks

**Comment by Drive4ik** [comment]
@grahamperrin 
Added a page that shows addons that have conflicts with STG.

@Mek101 Unfortunately no. If you make such an option, then the user by enabling this option and putting another addon will not know that he may have problems. Binding to the extension ID and not showing a conflict window for it won't work either. Firstly, users very often click "ok" without even reading the contents. Second, these settings must be constantly stored, transferred and restored. This makes absolutely no sense. I added to the list only those extensions, which disconnect the group and the tabs, and they can not work in any other way. For example, OneTab does not take hidden tabs into account at all. Using such addons together with STG brings more problems than good.
